src/pages/contact/index.tsx

```diff
@@ -0,0 +1,298 @@
+import React from 'react';
+import { Send } from 'react-feather';
+import { Helmet } from 'react-helmet';
+import { Box, Container, makeStyles } from '@material-ui/core';
+import { Field, Form, Formik } from 'formik';
+import { fieldToTextField, TextFieldProps } from 'formik-material-ui';
+import * as Yup from 'yup';
+
+import { Button } from '../../components/Button';
+import { DetailsCard } from '../../components/DetailsCard';
+import { Navbar } from '../../components/Navbar';
+import { SectionTitle } from '../../components/SectionTitle';
+import { TextField } from '../../components/TextField';
+import { useDeveloperProfile } from '../../containers/DeveloperProfile';
+import { useComponentType } from '../../hooks/useComponentType';
+import { textFieldData } from '../../views/contact-page/data';
+
+const contactPageItemShadow = '0 40px 50px 0 rgba(103, 118, 128, 0.1)';
+
+function TextFieldWrap(props: TextFieldProps) {
+  const {
+    form: { setFieldValue },
+    field: { name },
+  } = props;
+
+  const onChange = React.useCallback(
+    (event) => {
+      const { value } = event.target;
+      setFieldValue(name, value);
+    },
+    [setFieldValue, name],
+  );
+  return <TextField {...fieldToTextField(props)} {...textFieldData} onChange={onChange} />;
+}
+
+interface FormValues {
+  fullName: string;
+  email: string;
+  title: string;
+  messageContent: string;
+}
+
+const initialValues: FormValues = {
+  fullName: '',
+  email: '',
+  title: '',
+  messageContent: '',
+};
+
+const validationSchema = Yup.object().shape({
+  fullName: Yup.string().required('Full Name is required field'),
```

Comment:
Use `.label()` to provide the field's name so it wouldn't be necessary to add a custom message